Web_Development/Javascript/Arrow Functions/Readme.md

```diff
@@ -0,0 +1,72 @@
+# Arrow Functions in javascript
+<hr>
+##What are arrow functions?
```

Comment:
Give space between # and the content to get the results

Reply:
Made the changes, thanks for the feedback